Cargo.toml

```
@@ -8,7 +8,10 @@ name = "mio"
 # - Create git tag
 version       = "0.7.6"
 license       = "MIT"
-authors       = ["Carl Lerche <me@carllerche.com>"]
+authors       = [
+  "Carl Lerche <me@carllerche.com>",
+  "Thomas de Zeeuw <thomasdezeeuw@gmail.com>",
```

Fine w/ me :+1:. Could you also add "Tokio Contributors" to match https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml#L13

```suggestion
  "Thomas de Zeeuw <thomasdezeeuw@gmail.com>",
  "Tokio Contributors <team@tokio.rs>",
```